admin/pre_reg/utils.py

```diff
@@ -0,0 +1,36 @@
+
+SORT_BY = {
```

Comment:
Why is this necessary? Couldn't the client just pass `-initiator` instead of `n_initiator`?